packages/core/test/tracing/index_test.js

```diff
@@ -67,21 +78,27 @@ mochaSuiteFn('[UNIT] tracing/index', function () {
   });
 
   it('deactivate instrumentation via config', () => {
-    initAndActivate({ tracing: { disabledTracers: ['grpc', 'kafkajs'] } });
+    initAndActivate({ tracing: { disabledTracers: ['grpc', 'kafkajs', 'aws-sdk/v2/index'] } });
     expect(activateStubGrpc).to.not.have.been.called;
     expect(activateStubGrpcJs).to.have.been.called;
     expect(activateStubKafkaJs).to.not.have.been.called;
     expect(activateStubRdKafka).to.have.been.called;
     expect(initStubGrpc).not.to.have.been.called;
+
+    expect(initAwsSdkv2).not.to.have.been.called;
+    expect(activateAwsSdkv2).not.to.have.been.called;
```

Comment:
_suggestion:_ For some of the disabled instrumentations, we only check whether their activate has or has not been called, for others we check this for init as well as activate. Then, we also have a separate test case 'skip init step when instrumentation disabled' that checks all the init methods. 

Maybe we should merge this all into this test case to make it a bit more consistent? Something like this:

```
it('deactivate instrumentation via config', () => {
  initAndActivate({ tracing: { disabledTracers: ['grpc', 'kafkajs', 'aws-sdk/v2/index'] } });
  // grpc instrumentation  has been disabled, make sure neither its init nor its activate method are called
  expect(initStubGrpc).not.to.have.been.called;
  expect(activateStubGrpc).to.not.have.been.called;
  
  // grpcJs instrumentation has not been disabled, make sure its init and activate are called
  expect(initStubGrpcJs).to.have.been.called;
  expect(activateStubGrpcJs).to.have.been.called;
  
  // kafkajs has been disabled...
  expect(initStubKafkaJs).to.not.have.been.called;
  expect(activateStubKafkaJs).to.not.have.been.called;
      
  // ...but rdkafka has not been disabled
  expect(initStubRdKafka).to.have.been.called;
  expect(activateStubRdKafka).to.have.been.called;
  
  // aws-sdk/v2 has been disabled (via aws-sdk/v2/index)
  expect(initAwsSdkv2).not.to.have.been.called;
  expect(activateAwsSdkv2).not.to.have.been.called;
});
```

And then remove the separate test case 'skip init step when instrumentation disabled'.

Of course, this is not directly related to the changes in this PR.

And since you probably want to specifically test for `'aws-sdk/v2/index'` as well as `'aws-sdk/v2'`, we could have two separate test cases, one with the string with `/index` and one for the string without `/index`.